homeassistant/components/dynalite/__init__.py

```diff
@@ -70,24 +63,23 @@
 async def async_setup_entry(hass, entry):
     """Set up a bridge from a config entry."""
     LOGGER.debug("__init async_setup_entry %s", entry.data)
-    host = entry.data[CONF_HOST]
-    config = hass.data[DOMAIN][DATA_CONFIGS].get(host)
 
-    if config is None:
-        LOGGER.error("__init async_setup_entry empty config for host %s", host)
-        return False
-
-    bridge = DynaliteBridge(hass, entry)
+    bridge = DynaliteBridge(hass, entry.data[CONF_HOST])
 
     if not await bridge.async_setup():
         LOGGER.error("bridge.async_setup failed")
         return False
     hass.data[DOMAIN][entry.entry_id] = bridge
+
+    hass.async_create_task(
+        hass.config_entries.async_forward_entry_setup(entry, "light")
+    )
     return True
 
 
 async def async_unload_entry(hass, entry):
     """Unload a config entry."""
     LOGGER.error("async_unload_entry %s", entry.data)
```

Comment:
Please remove this logging or set level to debug.

Reply:
sorry. leftover from initial tests. removed